grabber/main.go

```diff
@@ -98,7 +98,7 @@ func main() {
 	}
 	err := app.Run(os.Args)
 	if err != nil {
-		log.Fatal().Err(err).Msg("Run")
+		fmt.Fprintf(os.Stderr, "Failed to start app: %v\n", err)
```

Comment:
How about just `Fatal error`, since this could happen at any point, not just after starting.
This should also go through the logger - b/c anything that doesn't risks silently being dropped, or at least overlooked by not being structured correctly (e.g. we lose severity).